WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -231,6 +273,13 @@ protected void next(String email) {
     public void onDetach() {
         super.onDetach();
         mLoginListener = null;
+
+        if (mGoogleApiClient.isConnected()) {
+            mGoogleApiClient.stopAutoManage(getActivity());
+            mGoogleApiClient.disconnect();
+        }
+
+        ActivityUtils.showKeyboard(getActivity().getCurrentFocus());
```

Comment:
Hmm, can you elaborate on the need for resuming the keyboard here?

Reply:
I don't know why that is there.  I remove it in fbf72d69e2c7b8a824c309220a285322e5056c80.